pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -72,6 +73,8 @@ const (
 	}
 }
 `
+	// temporary until fix org isolation
```

Comment:
```suggestion
	//TODO: temporary until fix org isolation
```

Reply:
I suggest better adding this as a part of #37320
In this one I have focused on the DB modifications and I have introduced only the necessary Alertmanager modification so that tests and the server run without failures for the main organisation.